config/secrets.yml

```diff
@@ -1,5 +1,7 @@
 common: &common
   secret_key_base: 'f0b774b4d06f7ea3357dec260cd6231367009d35525b3c437b27e299d52599bebd4eb1cd83fc3f69ff0d46a0510e5e4ba3bffd26b503b7713100f6083fc4baba'
+  auth_secret_key: '89e115653bae4fafda3fc4c45b64aa07e20ad4d75ab4c1637e85e44b50e4a5cd24ea4dc065e047642d2ed17bebae7e856a292ab1f46ff7d04fc473df1a9ac577'
```

Comment:
It is a command to the reader.

Often the pattern in secrets/config.yml is `value: <%= ENV['NAME'] || 'default' %>` but with these secrets using that pattern is unsafe as rails would happily work but would use a publicly known value for `secret_key_base`.